docs/API.md

```diff
@@ -296,6 +298,9 @@ Returns:
 
 ## .emailRecord(emailBuffer) ##
 
+Note: Please use `.accountRecord` instead of this method. This method only reads from the account table whereas 
+`.accountRecord` checks the emails table and returns correct account record.
```

Comment:
I don't have all the context here, sorry, but is the plan to remove this method eventually? When can we safely remove it? Should we back this comment up with a call to [`util.deprecate`](https://nodejs.org/api/util.html#util_util_deprecate_function_string) in the code itself?